client/src/pages/Launch.js

```diff
@@ -59,11 +60,16 @@ const Logo = styled.img`
 `;
 
 export const Launch = () => {
+  const history = useHistory();
   return (
     <Wrapper>
       <Title>DispoDisco</Title>
       <LogoContainer>
-        <Logo src={logoDark} alt="Logo Dark" />
+        <Logo
+          src={logoDark}
+          alt="Logo Dark"
+          onClick={() => history.push("/menu")}
+        />
```

Comment:
Is there an advantage using `history.push` over `<Link>`? 
For navigation items, it's preferable to use `<Link>`.

Reply:
Decided to use `history.push` over link, so we don't have to wrap our image into a link. Same goes for the buttons. That way we could just use our button component without having to translate it into a  `<Link>`.

Considering to change it in a later Update.